src/plugins/condition/criterion/TelemetryCriterion.js

```diff
@@ -76,10 +76,8 @@ export default class TelemetryCriterion extends EventEmitter {
         let params = [];
         let result = false;
         params.push(data[this.metadata]);
-        if (this.input instanceof Array && this.input.length) {
-            params.push(this.input[0]);
-        } else if (this.input) {
-            params.push(this.input);
+        if (this.input.length) {
```

Comment:
I think we should keep the `this.input instanceof Array` check - it doesn't hurt. It bothers me that we assume that this.input exists before checking for it's length property.